src/main/java/com/habitmon/domain/habitmon/api/HabitMonController.java

```diff
@@ -4,25 +4,38 @@
 import com.habitmon.common.response.ApiResponse;
 import com.habitmon.domain.habitmon.api.dto.request.HabitMonCreateRequest;
 import com.habitmon.domain.habitmon.api.dto.response.HabitMonCreateResponse;
+import com.habitmon.domain.habitmon.api.dto.response.HabitMonInfoResponse;
 import com.habitmon.domain.habitmon.service.HabitMonService;
+import com.habitmon.global.auth.AuthDetails;
 import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 @RequiredArgsConstructor
-@RequestMapping("/habit")
+@RequestMapping("/habitMon")
```

Comment:
RESTful 규격에 맞춰서 전부 소문자로 바꿔주시면 좋을 것 같아요~!
```suggestion
@RequestMapping("/habitmon")
```